src/validators/function.rs

```diff
@@ -306,6 +306,8 @@ pub struct ValidationInfo {
     config: PyObject,
     #[pyo3(get)]
     context: Option<PyObject>,
+    #[pyo3(get)]
+    field: Option<Py<PyString>>,
```

Comment:
```suggestion
    field: Option<String>,
```
make this is a `String`, then add a `getter` method  to the py implementation to get a `String`.

Or you might just be able to leave this as 

```rs
    #[pyo3(get)]
    field: Option<String>,
```
and let pyo3 take care of lazyily creating the `PyString`.

Reply:
This, and the clone which will be required either way, is a good argument for making the info argument truly optional.